forms/TextArea/index.js

```diff
@@ -27,7 +27,7 @@ export default React.createClass({
     mask: React.PropTypes.func,
     onFocus: React.PropTypes.func,
     onChange: React.PropTypes.func,
-    validate: React.PropTypes.func,
+    validate: React.PropTypes.oneOfType([React.PropTypes.func, React.PropTypes.array]),
```

Comment:
I think it's `oneOfType([React.PropTypes.func, React.PropTypes.string, React.PropTypes.array])`

Also, "func" works in the new system but it's expected to be a composed validator, or to be configured to call the callbacks correctly. This was the case before, but a lot of the users weren't calling the callback properly but were instead hacking themselves in via the errorMessage param.

String/Array are safer options because they keep the complexity wrapped up in validation.compose.